src/EditorFeatures/Core/InlineRename/AbstractEditorInlineRenameService.SymbolRenameInfo.cs

```diff
@@ -211,7 +211,7 @@ public bool TryOnAfterGlobalSymbolRenamed(Workspace workspace, IEnumerable<Docum
             public InlineRenameFileRenameInfo GetFileRenameInfo()
             {
                 if (RenameSymbol.Kind == SymbolKind.NamedType &&
-                    _document.Project.Solution.Workspace.CanApplyChange(ApplyChangesKind.ChangeDocumentInfo))
+                    _document.Project.Solution.CanApplyChange(ApplyChangesKind.ChangeDocumentInfo))
```

Comment:
query/transformation APIs that operate just on snapshots can now determine what changes the snapshot supports without going back to the workspace directly.